### PR TITLE
Make navbar scrollable and add info that fallback layer is excluded

### LINKED ISSUE #4342: Give min-width to navbar (overflow-x: auto)
On very small screens, elements are overflowing. Instead, the navbar could be scrollable.

### PR DESCRIPTION
**Pull request by MichaelBuessemeyer**
This PR makes the navbar scrollable for smalls screens as otherwise button group form weird multiline looking constructs.
As the items and their size in the navbar itself are variable, I chose to give the button groups a `min-width` depending on their size, forcing them to stay inline.

## TODO
 - [x] Check the navbar while not in a tracing: Does it look fine on small screens? -> Should be fine

### URL of deployed dev instance (used for testing):
- https://makenavbarscrollable.webknossos.xyz

### Steps to test:
- Open the dashboard and resize the window
- Open a tracing and resize the window
- In both cases, the responsiveness should have improved

### Issues:
- fixes #4342, #4394

------
- [x] Updated [changelog](../blob/master/CHANGELOG.md#unreleased)
- ~~[ ] Updated [migration guide](../blob/master/MIGRATIONS.md#unreleased) if applicable~~
- ~~[ ] Updated [documentation](../blob/master/docs) if applicable~~
- ~~[ ] Adapted [wk-connect](https://github.com/scalableminds/webknossos-connect) if datastore API changes~~
- ~~[ ] Needs datastore update after deployment~~
- [x] Ready for review

### CODE REVIEW

**Comment by MichaelBuessemeyer** [comment]
@daniel-wer Could you please review this PR?

Please note that I looked for a better solution regarding the scrollablitity of the navbar then giving it a static `min-width`. 
The core problem is that the button group in the navbar breaks into multiple lines when the screen width is too small. That's why it is enough to give the wrapper of this group a mind-width. Sadly the width of this group varies. Thus I wrote a small calculation that calculates the necessary `min-width` depending on the variation of the button group. I hope that's fine :)

File: frontend/javascripts/navbar.js [hunk 1]
```diff
@@ -294,6 +294,7 @@ function Navbar({ activeUser, isAuthenticated, history, isInAnnotationView, hasO
 
   const navbarStyle: Object = {
     padding: 0,
+    overflowX: "auto",
```
Comment (daniel-wer) [inline frontend/javascripts/navbar.js]: I've got some good news, adding `white-space: nowrap` here will make all of the calculations and the `min-width` in the `tracing_actions_view.js` obsolete :)

**Comment by MichaelBuessemeyer** [comment]
@daniel-wer Could you please review this pr again?

**Pull request merged**